package.json

```
@@ -68,6 +68,7 @@
     "@types/react-modal": "3.16.3",
     "@types/react-router-dom": "5.3.3",
     "@typescript-eslint/parser": "7.0.2",
+    "buffer": "^6.0.3",
```

Can we pin this to an exact version? We've had more than a few bugs related to minor level updates
```suggestion
    "buffer": "6.0.3",
```